Tests/OptimizelyTests-MultiClients/AtomicDictionaryTests.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright 2021, Optimizely, Inc. and contributors 
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");  
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at   
+// 
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import XCTest
+
+class AtomicDictionaryTests: XCTestCase {
```

Comment:
Can we have a test where we add atomic dictionaries to atomic array and/or vice versa? just want to see if there are any issues if used together.

Reply:
Interesting. It's expected to work ok, but I won't add testing cases since we do not have a plan to use them so extensively :)